docs-ui/components/text.stories.js

```diff
@@ -1,80 +1,37 @@
 import React from 'react';
 import {withInfo} from '@storybook/addon-info';
 
-import Text from 'app/components/text';
-
 export default {
   title: 'Core/Style/Text',
 };
 
 export const Default = withInfo('On-demand styling for native dom elements')(() => (
   <div style={{padding: 20, backgroundColor: '#ffffff'}}>
-    <Text>
-      <h1>Text styles</h1>
-      <p>
-        Not having to override margin, padding, and line-height on native text elements is
-        a feature in the app UI world — that's why things like reset.css, normalize.css,
-        etc exist. But in situations where you need those styles to "just work" it can be
-        quite frustrating to find out they're not supported.
-      </p>
```

Comment:
These docs need to be updated and improved. Ignoring this for now but I’ll come back to it.